server/cp/output/formatter/jimi.py

```diff
@@ -0,0 +1,79 @@
+
+import superdesk
+import lxml.etree as etree
+
+from superdesk.utc import utc_to_local
+from superdesk.text_utils import get_text
+from superdesk.publish.formatters import Formatter
+
+
+class JimiFormatter(Formatter):
+
+    ENCODING = 'utf-8'
+
+    def can_format(self, format_type, article):
+        return format_type == 'jimi'
+
+    def format(self, article, subscriber, codes=None):
+        pub_seq_num = superdesk.get_resource_service('subscribers').generate_sequence_number(subscriber)
+        root = etree.Element('Publish')
+        self._format_item(root, article)
+        xml = etree.tostring(root, pretty_print=True, encoding=self.ENCODING, xml_declaration=True)
+        return [(pub_seq_num, xml.decode(self.ENCODING))]
+
+    def _format_item(self, root, item):
+        content = etree.SubElement(root, 'ContentItem')
+
+        # root system fields
+        etree.SubElement(root, 'Reschedule').text = 'false'
+        etree.SubElement(root, 'IsRegional').text = 'false'
+        etree.SubElement(root, 'CanAutoRoute').text = 'true'
+        etree.SubElement(root, 'PublishID').text = '1'
+        etree.SubElement(root, 'Services').text = 'Print'
+        etree.SubElement(root, 'Username')
+        etree.SubElement(root, 'UseLocalsOut').text = 'false'
+        etree.SubElement(root, 'PscCodes').text = 'ap---'
+
+        # content system fields
+        etree.SubElement(content, 'Name')
+        etree.SubElement(content, 'Cachable').text = 'false'
+        etree.SubElement(content, 'NewsCompID').text = '2'
+
+        # timestamps
+        firstpublished = item.get('firstpublished') or item['versioncreated']
+        etree.SubElement(root, 'PublishDateTime').text = self._format_datetime(firstpublished)
+        etree.SubElement(content, 'EmbargoTime').text = '0001-01-01T00:00:00'
```

Comment:
is it a correct value?

Reply:
that seems to be the default which means not set or something